elm-antd-extras/index.js

```diff
@@ -0,0 +1,10 @@
+const buttons = document.querySelectorAll('.animated-btn');
```

Comment:
Use the following convention for classes:

`elm-antd__<snake_case_identifier>`. 

So in this case it would be, `elm-antd__animated_button`